src/main/java/com/zaxxer/sansorm/internal/Introspected.java

```diff
@@ -29,23 +31,21 @@
 import java.util.Arrays;
 import java.util.Collection;
 import java.util.HashMap;
-import java.util.LinkedHashMap;
 import java.util.LinkedList;
 import java.util.List;
 import java.util.Map;
 import java.util.Map.Entry;
+import java.util.TreeMap;
 
 import javax.persistence.*;
 
-import org.postgresql.util.PGobject;
```

Comment:
i'm still can't figure out proper import config for entire project - it is just inconsistent